app/containers/daedalusTransfer/DaedalusTransferPage.js

```diff
@@ -125,6 +131,9 @@ export default class DaedalusTransferPage extends Component<InjectedProps> {
           </MainLayout>
         );
       case 'readyToTransfer':
+        if (daedalusTransfer.transferTx == null) {
```

Comment:
I think this one should be `===` because it seems that is never `undefined`

Reply:
getting:
```
[flow] Cannot create `DaedalusTransferSummaryPage` element because null or undefined [1] is incompatible with `TransferTx` [2] in property `transferTx`. (DaedalusTransferPage.js:141:27)
```
on line#141 if i change to `===`